checkpoint/cp-resource-management/templates/configmap.yaml

```diff
@@ -1,14 +1,14 @@
-{{ if .Values.clusterID }}
-kind: ConfigMap 
-apiVersion: v1 
-metadata:
-  annotations:
-    "helm.sh/hook": "pre-install"
-  name: cp-dome9-configmap
-  labels:
-    app.kubernetes.io/name: {{ template "cp-resource-management.name" . }}
-    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-data:
-  clusterID: {{ .Values.clusterID | quote }}
-{{ end }}
+{{ if .Values.clusterID }}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  annotations:
+    "helm.sh/hook": "pre-install"
+  name: {{ .Release.Name }}-cp-cloudguard-configmap

```

Comment:
will influence upgrades. just need to be documented, unless we want to copy configmap for the customer (feasible? not sure it worth with small numbers of customers we have but..)

Reply:
This is still a problem and some upgrades will break due to issues with Helm - configmap isn't upgraded or recreated correctly when you make changes to the name and/or content of the configmap.
There is an open issue with Helm on this  - [#5915](https://github.com/helm/helm/issues/5915) and a couple of others.
Hopefully this will be fixed once helm correctly implements the '--force' flag and adds a '--create' flag

Reply:
so we expect customers to delete and deploy instead of upgrade?